worlds/bomb_rush_cyberfunk/__init__.py

```diff
@@ -0,0 +1,212 @@
+from typing import Any, Dict
+from BaseClasses import MultiWorld, Region, Location, Item, Tutorial, ItemClassification
+from worlds.AutoWorld import World, WebWorld
+from .Items import base_id, item_table, group_table, BRCType
+from .Locations import location_table, event_table
+from .Regions import region_names, region_exits, BRCStage
+from .Rules import rules
+from .Options import BombRushCyberfunkOptions, StartStyle
+
+
+class BombRushCyberfunkWeb(WebWorld):
+    theme = "ocean"
+    tutorials = [Tutorial(
+        "Multiworld Setup Guide",
+        "A guite to setting up Bomb Rush Cyberfunk randomizer and connecting to an Archipelago Multiworld",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["TRPG"]
+    )]
+
+
+class BombRushCyberfunkWorld(World):
+    """Bomb Rush Cyberfunk is 1 second per second of advanced funkstyle. Battle rival crews and dispatch militarized 
+    police to conquer the five boroughs of New Amsterdam. Become All City."""
+
+    game = "Bomb Rush Cyberfunk"
+    web = BombRushCyberfunkWeb()
+    data_version = 1
```

Comment:
```suggestion
```
Data version is deprecated, and there's been talks about fully removing it. Would recommend deleting this line.

Reply:
That's good to know